tasks/main.yml

```diff
@@ -9,7 +9,18 @@
   when: ansible_os_family == "Suse"
 
 - include: agent5.yml
-  when: datadog_agent5
+  when: datadog_agent5 and ansible_os_family != "Windows"
```

Comment:
Could we add the `-linux` suffix to `agent5.yaml` and `agent6.yaml` files ?

Reply:
I was going for minimal change.

Reply:
I think a rename is fine and make things more explicit